utils/litemap/src/map.rs

```diff
@@ -219,3 +219,32 @@ impl<K, V> LiteMap<K, V> {
         self.values.iter_mut().map(|val| (&val.0, &mut val.1))
     }
 }
+
+#[cfg(feature = "serde")]
+impl<K: serde::Serialize, V: serde::Serialize> serde::Serialize for LiteMap<K, V> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        self.values.serialize(serializer)
+    }
+}
```

Comment:
Can you derive here? It seems like it should work.

Reply:
I just didn't want to introduce a derive dependency. Best to avoid `syn` as much as possible.

Reply:
Ah, good point!